Lib/asyncio/queues.py

```diff
@@ -41,6 +58,7 @@ def __init__(self, maxsize=0):
         self._finished = locks.Event()
         self._finished.set()
         self._init(maxsize)
+        self.shutdown_state = _queue_alive
```

Comment:
A rewrite with the enum:
```suggestion
        self._shutdown_state = _QueueState.ALIVE
```